modules/reference/pages/cluster-properties.adoc

```diff
@@ -1638,6 +1638,20 @@ Target replication factor for internal topics.
 
 ---
 
+=== minimum_topic_replications
+
+Minimum permitted replication factor value for newly created topics. Redpanda issues an error for any attempt to create a topic with a replication factor less than the specified value. Any existing topics found to have a replication factor less than the specified value will still continue to work as normal. Redpanda will issue a warning level log message at start up indicating the topic's replication factor is less than the specified minimum.
```

Comment:
@michael-redpanda what is the Kafka API error code/message that we return when this is violated at topic creation?  Would be good to document this.

Also what is the log message when exiting topics violate it.  Would be good to add this detail for users to follow. @greg-szrama

Reply:
+1 on documenting the msgs here.

Reply:
> what is the Kafka API error code/message that we return when this is violated at topic creation? Would be good to document this.

The rc is "INVALID_REPLICATION_FACTOR" (38) - message is "Replication factor must be greater than or equal to specified minimum value"

> Also what is the log message when exiting topics violate it.
warning level on the cluster logger facility: "Topic X has a replication factor less than specified minimum: {} < {}"

Should be printed on startup and whenever the config value changes.